### PR TITLE
Feature/tinymce5update 3 minor fixes

### PR DESCRIPTION
**Pull request by lorenzo-pomili**
Related Ticket: 

Description of Changes:
this PR contains a couple of minor fixes

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/pages/advanced/security.adoc [hunk 1]
```diff
@@ -127,7 +127,7 @@ Cross-Site Scripting (XSS) involves using website or application inputs to injec
 
 Although {productname} filters content such as scripts from the editor content, precautions should be taken to protect your users, such as enabling a Content Security Policy (CSP) and server-side filtering.
 
-For information on Cross-Site Scripting and how to reduce the risk of an attack, see: https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A7-Cross-Site_Scripting_(XSS)[OWASP Top Ten 2017 -- Cross-Site Scripting (XSS)].
+For information on Cross-Site Scripting and how to reduce the risk of an attack, see: https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A7-Cross-Site_Scripting_(XSS)[OWASP Top Ten 2017 _-_ Cross-Site Scripting (XSS)].
```
Comment (lnewson) [inline modules/ROOT/pages/advanced/security.adoc:130]: This was correct previously as this is meant to be an em dash.

- [ ] Revert the changes to this file for `--`

File: modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc [hunk 2]
```diff
@@ -1,7 +1,9 @@
+:blank: blank_
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc:1]: Can we please call this `blanktarget` instead to match what was used in the 6.x docs? https://raw.githubusercontent.com/tinymce/tinymce-docs/staging/docs-6/modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/pages/advanced/security.adoc (hunk 1 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/advanced/security.adoc]: done

File: modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc (hunk 2 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc]: done

**Review by ltrouton — APPROVED** [review]

**Review by metricjs — APPROVED** [review]
